backbone.js

```diff
@@ -979,14 +979,14 @@
     sort: function(options) {
       if (!this.comparator) throw new Error('Cannot sort a set without a comparator');
       options || (options = {});
+      var comparator = _.isFunction(this.comparator) ? _.bind(this.comparator, this) : this.comparator;
```

Comment:
Sure but that sort of defeats the purpose of `iteratee` which is supposed to sugar a function. It forces devs to do `_.isFunction(iter) ? _.bind(iter, context) : _.iterate(iter);` or for them to override `iteratee`. Seems like a low hanger for a common case